### DISCUSSION TITLE
Mocap Calibration with drone

### DISCUSSION THREAD

**Discussion by salooooo** [bitcraze/discussions#617] [Q&A]
Hey everyone, I have a huge volume, (30x55m) that is covered using a Qualisys system. The calibration for this entire volume takes around ~5-10min using the provided wand and can still be inaccurate in some locations. Is it possible to use the 2.x drone to calibrate this volume for  measurements? I'm not sure if it's possible but I did hear about the connectivity and was wondering if any sort of this kind of solution was possible using either the active marker or passive marker deck for the drone. Thanks!

**Comment by krichardsson** [comment]
I know this idea has been discussed at Qualisys, but as far as I know it has not been implemented. I'm not sure the distance between the markers on the marker decks are long enough, or if the precision would be enough, you should probably check with Qualisys. 

It could be an interesting functionality also if your space is high, I guess it is hard to reach with the calibration wand high up to calibrate the full space.

**Reply by salooooo** [reply]
For sure. I'm just trying to calibrate the lower half of the volume at the moment, (I'll be getting some kind of pipe to extend the pole once I can get consistent calibrations), I'll try reaching out to them and see what they say.
